src/main/java/com/chaos131/gamepads/Gamepad.java

```diff
@@ -6,11 +6,24 @@
 
 import edu.wpi.first.wpilibj2.command.button.CommandXboxController;
 import edu.wpi.first.math.MathUtil;
+import edu.wpi.first.math.filter.SlewRateLimiter;
 
 
 /** A CHAOS Wrapper around `CommandXboxController` */
 public class Gamepad extends CommandXboxController {
+    private SlewRateLimiter m_slewratelimiterLeftX;
+    private SlewRateLimiter m_slewratelimiterLeftY;
+    private SlewRateLimiter m_slewratelimiterRightX;
+    private SlewRateLimiter m_slewratelimiterRightY;
+   
 
+    public Gamepad(int port, double value, double rotRate) {
```

Comment:
Can we change the names of these to leftStickSlewRate and rightStickSlewRate? Or something like this?

And maybe add a javadoc comment explaining this (we can have some students do this tomorrow)